test/test_object.py

```diff
@@ -10,6 +10,48 @@
 
 
 class TestObject(HPyTest):
+    def test_getattr(self):
+        import pytest
+        mod = self.make_module("""
+            HPy_DEF_METH_O(f)
+            static HPy f_impl(HPyContext ctx, HPy self, HPy arg)
+            {
+                HPy name, result;
+                name = HPyUnicode_FromString(ctx, "foo");
+                if (HPy_IsNull(name))
+                    return HPy_NULL;
+                result = HPy_GetAttr(ctx, arg, name);
+                HPy_Close(ctx, name);
+                if (HPy_IsNull(result))
+                    return HPy_NULL;
+                return result;
+            }
+            @EXPORT f HPy_METH_O
+            @INIT
+        """)
+
+        class Attrs:
```

Comment:
good idea to use many different kinds of attrs to test! 👍👍👍

Reply:
:D